File: HelloWorld/HelloWorld.java
import java.util.*;

class HelloWorld {
  static final List<String> CENSUS = new ArrayList<String>();

  static {
    /*
     * Add yourself using the following format:
     *    CENSUS.add("Your Name @ https://github.com/username");
     */
    CENSUS.add("Allen Comp Sci @ https://github.com/AllenCompSci");
    CENSUS.add("Mr. Hudson @ https://github.com/theshrewedshrew");
    CENSUS.add("BEST Team 58 @ https://github.com/BESTTeam58");
    CENSUS.add("Texas Snow @ https://github.com/TexasSnow");
    CENSUS.add("Rahultheman12 @ https://github.com/Rahultheman12");
    CENSUS.add("John Nguyen @ https://github.com/jawnlovesfreestuff");
    CENSUS.add("Carson Mires @ https://github.com/carsonmires");
    CENSUS.add("Cameron Mathis @ https://github.com/Phylux");
    CENSUS.add("YourFriendlyNeighborhoodSpiderman @ https://github.com/YourFriendlyNeighborhoodSpiderman");
    CENSUS.add("Devin Petersen @ https://github.com/DevinPetersen");
    CENSUS.add("Kaushik Malapati@ https://github.com/KaushikMalapati");
    CENSUS.add("Kyleaustin36 @ https://github.com/kyleaustin36");
    CENSUS.add("Zak Ishak @ https://github.com/zakishak");
    CENSUS.add("Maaz Kamal @ https://github.com/Maze-Camel");
    CENSUS.add("Justin Van Nimwegen @ https://github.com/JustinV10");
    CENSUS.add("bingood4ever @ https://github.com/bingood4ever");
    CENSUS.add("Gainz101 @ https://github.com/Gainz101");
    CENSUS.add("zachdogg @ https://github.com/Zachdogg1");
    CENSUS.add("Johnathan G. @ https://github.com/SodaCan0515");
    CENSUS.add("MJHudson618 @ https://github.com/MJHudson618");
    CENSUS.add("Abhi1458 @ https://github.com/Abhi1458");
    CENSUS.add("Maxwell Cody @ https://github.com/MaxwellCody");
  }

  public static void main(String[] args) {
    for(String entry : CENSUS) {
        System.out.println("Hello World from " + entry);
    }
  }
}

File: HelloWorld/HelloWorld.c
#include <stdio.h>
#include <stdlib.h>

void add_entry(char *, char *);
void populate_census();

struct census_entry {
   char * fullName;
   char * githubUsername;
   struct census_entry * next;
};

typedef struct census_entry CensusEntry_t;

static CensusEntry_t * head = NULL;
static CensusEntry_t * tail = NULL;

void populate_census() {
   /*
    * ADD YOURSELF HERE!
    *
    * Add yourself using the following format:
    *    add_entry("Your Name", "github_username");
    */
   add_entry("Allen Comp Sci", "AllenCompSci");
   add_entry("Maxwell Cody", "MaxwellCody");
}

int main(int argc, char * argv[]) {
   populate_census();

   for(CensusEntry_t * current = head; current != NULL; current = current->next) {
      printf("Hello World from %s @ https://github.com/%s\n", current->fullName, current->githubUsername);
   }

   return EXIT_SUCCESS;
}

void add_entry(char * fullName, char * githubUsername) {
   CensusEntry_t * entry = malloc(sizeof(CensusEntry_t));
   entry->fullName = fullName;
   entry->githubUsername = githubUsername;
   entry->next = NULL;

   // If one of these conditions is true, then they both should be
   // we're checking both just to be sure.
   if(head == NULL || tail == NULL) {
      head = entry;
   } else {
      tail->next = entry;
   }
   tail = entry;
}
